newsroom/agenda/formatters/csv_formatter.py

```diff
@@ -27,21 +28,21 @@ def format_items(self, items, item_type=None):
             f"{datetime.now().strftime('%Y-%m-%d-%H:%M:%S')}-{'multi'}.{self.FILE_EXTENSION}"
         )
 
-    def serialize_to_csv(self, data):
+    def serialize_to_csv(self, items: Union[List[Dict[str, Any]], Dict[str, Any]]) -> bytes:
```

Comment:
imo it would be better to have this work with just list, you can call it from `format_event` with `[event]`, that would simplify the types and code in general